typings/index.d.ts

```diff
@@ -105,24 +105,29 @@ export interface GetInputPropsOptions
 export interface GetLabelPropsOptions
   extends React.HTMLProps<HTMLLabelElement> {}
 
-export interface getToggleButtonPropsOptions
+export interface GetToggleButtonPropsOptions
   extends React.HTMLProps<HTMLButtonElement> {}
 
-interface OptionalExtraGetItemPropsOptions {
-  [key: string]: any
+export interface GetMenuPropsOptions<Item>
+  extends Record<string, any> {
```

Comment:
This doesn't seem to be quite right. I also don't understand what the `<Item>` generic type is doing here as it doesn't appear to be used.

From looking over the docs, this is what I would expect this type to look like:
```
export interface GetMenuPropsOptions {
  refKey?: string
  aria-label?: string
}

Reply:
You are completely right. I kind of copied the interface from `GetMenuItemOptions` which has the generic type parameter and I forgot to remove it.